charts/icm-as/templates/_probes.tpl

```diff
@@ -12,9 +12,12 @@ startupProbe:
   initialDelaySeconds: {{ .Values.probes.startup.initialDelaySeconds | default 60 }}
   periodSeconds: {{ .Values.probes.startup.periodSeconds | default 10 }}
 livenessProbe:
-  httpGet:
-    path: /status/LivenessProbe
-    port: mgnt
+  exec:
+    command:
+      - /bin/bash
+      - -c
+      - |
+        [ -f "/tmp/liveness-status" ] || curl -f http://localhost:{{ .Values.ports.mgnt }}/status/LivenessProbe >/dev/null 2>&1
```

Comment:
May add a comment with the link to the originating issue in the K8s repository for future reference:
https://github.com/kubernetes/kubernetes/issues/57187